modules/ROOT/pages/manage-monitor-applications.adoc

```diff
@@ -0,0 +1,35 @@
+= Monitoring Applications Deployed to Runtime Fabric
+
+Anypoint Platform provides multiple applications that enable you to monitor applications deployed to Runtime Fabric.
+
+== Anypoint Monitoring
+
+xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components in your application network. For applications deployed to Runtime Fabric, Anypoint monitoring provides basic metrics to all customers. Titanium customers have access to advanced metrics and application-level logging.
+
+When you deploy an application to Runtime Fabric, Anypoint Monitoring is enabled by default. To disable Anypoint Monitoring, pass the following custom property when deploying your application:
+
+----
+anypoint.platform.config.analytics.agent.enabled=false
+----
+
+=== Required Resources
+
+Anpoint Monitoring creates metrics data at a consistent rate, normally one megabyte per hour.
+
+For application logs, the amount of memory required depends on the log level of your application. At the high end, and application can consume up to 400Mb.
```

Comment:
...an application (last sentence in line 19)

Reply:
Fixed in Sundip's PR.